assets/shared.css

```diff
@@ -35,6 +38,53 @@ a {
     color: var(--link-color);
     text-decoration: none;
     font-weight: bold;
+    transition: 250ms ease-in;
+}
+
+a:hover {
+    color: var(--link-active-color);
```

Comment:
I think there's no need to change color, it looks pretty good with just the underline. Not sure if the animation is then appropriate, if the color is unchanged.